models/ed/tests/testthat/test-read_ET_files.R

```diff
@@ -1,3 +1,4 @@
+source("../../R/model2netcdf.ED2.R")
```

Comment:
Do not source package files directly into test files -- testthat [already takes care of](https://r-pkgs.org/testing-basics.html#test-organisation) loading the package namespace, so loading them again can only cause confusion and differences between behavior under test and behavior at user run time.

```suggestion
```